lib/cfn-nag/custom_rules/passrole_base_rule.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+require 'cfn-nag/util/wildcard_patterns'
+
+class PassRoleBaseRule < BaseRule
+  IAM_ACTION_PATTERNS = wildcard_patterns('PassRole').map! { |x| 'iam:' + x } + ['*']
```

Comment:
two small nits here:

1. i don't believe you need to use map! vs. just map? or is there some technicality with this being a constant?  bang sometimes means there is a whacky side effect 
2. rename "x" as pass_role_with_asterisk or the like for the readability

Reply:
Put the ! because I thought that was needed to change the original array, but yeah thinking about it the assigning the results means the ! is not needed.   

Changed name to `pattern` which is consistent with naming in other locations using a map on a plural variable.